controllers/cheuser/namespacecache.go

```diff
@@ -0,0 +1,121 @@
+package cheuser
+
+import (
+	"context"
+	"sync"
+
+	"github.com/devfile/devworkspace-operator/pkg/infrastructure"
+	projectv1 "github.com/openshift/api/project/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	ownerUidLabel            string = "che.eclipse.org/workspace-namespace-owner-uid"
```

Comment:
To be consistent with const below
```suggestion
	cheClusterOwnerUidLabel            string = "che.eclipse.org/workspace-namespace-owner-uid"
```